variables.tf

```diff
@@ -20,12 +20,24 @@ variable "private_network" {
 # OPTIONAL PARAMETERS
 # ----------------------------------------------------------------------------------------------------------------------
 
-variable "user_name" {
+variable "default_user_name" {
   description = "The name of the default database user."
   type        = string
   default     = "default"
```

Comment:
This could be defaulted to **root** as well.

Reply:
Hi @sundarsudandiraraj I have made this change.